src/sbvr-api/permissions.ts

```diff
@@ -312,7 +312,7 @@ const namespaceRelationships = (
 	});
 };
 
-type PermissionLookup = Dictionary<true | string[]>;
+export type PermissionLookup = _.Dictionary<true | string[]>;
```

Comment:
just for my info, what is the difference between lodash Dictionary and pinejs-client-core Dictionary ?

Reply:
@JSReds good catch. thx! This is a mistake. I only needed to export the PermissionLookup and not change the type.
Fixed to not use lodash dictionary.